src/cloffeine/cache.clj

```diff
@@ -42,3 +43,20 @@
 
 (defn policy [^Cache cache]
   (.policy cache))
+
+(defn as-map ^ConcurrentMap [^Cache cache]
+  (.asMap cache))
+
+(defn- compute- [^ConcurrentMap m k bi-fn]
```

Comment:
`compute` is from the Map interface, so we don't really need to specify the implementation of the Map.